lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -78,6 +82,19 @@ module.exports = {
     };
   },
 
+  validateConditions(conditions, functionName) {
+    // At least one condition must be set
+    // https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenerrule.html#cfn-elasticloadbalancingv2-listenerrule-conditions
+
+    if (Object.getOwnPropertyNames(conditions).length > 0) {
+      return;
+    }
+
+    throw new this.serverless.classes.Error(
+      `At least 1 condition for alb event must be set in function "${functionName}".`
```

Comment:
Let's word it maybe as: `Cannot setup "alb" event in "${functionName}" function: At least one condition must be defined`